src/screens/FeaturedSearchScreen.tsx

```diff
@@ -8,10 +12,36 @@ import SearchInput from '../components/SearchInput';
  * @returns {JSX.Element} | 'not logged in' search screen, the landing page of the app
  */
 export default function FeaturedSearchScreen(): JSX.Element {
+    const { profile, setProfile } = useProfileContext();
+    const [loading, setLoading] = useState<boolean>(true);
+    const [trendingShows, setTrendingShows] = useState<ShowData[] | null>(null);
+
+    useEffect(() => {
+        const handler = async () => {
+            const movieData: ShowData[] | null = await getMovieTrending();
+            const tvData: ShowData[] | null = await getTvTrending();
+            if (movieData && tvData) {
+                setTrendingShows([...movieData, ...tvData]);
+            }
+            setLoading(false);
+        };
+        handler();
+    }, []);
```

Comment:
Out of scope for this PR, but we can likely make this a custom hook `useTrendingShows()` or something.

Reply:
Created #457